e2e/cypress/tests/integration/messaging/forward_message_from_dm_spec.js

```diff
@@ -0,0 +1,220 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
We're currently reorganizing folder structure in Zephyr where we are aligning with multi-product architecture. A structure like "Product > Feature > Test Cases". With that, please move spec files to its new folder like below so it will be easier for us once we move around spec files later on.
```
e2e/cypress/tests/integration/message_forwarding/
```